src/main/java/fi/tkgwf/ruuvi/config/Config.java

```diff
@@ -419,7 +419,7 @@ public static long getMeasurementUpdateLimit() {
     }
 
     public static boolean isAllowedMAC(String mac) {
-        return filterMode.test(mac);
+        return mac != null &&  filterMode.test(mac);
```

Comment:
Is there an extra space now or is github diff being wonky? 🤔

Reply:
Not github - its me being wonky :(